bin/zopen-importenvs

```diff
@@ -1,18 +1,87 @@
-#!/bin/sh
-#
+#!/bin/bash
 # Used for development to source all the boot .envs
-# . importenvs
+#. ./zopen-importenvs <path to buildenv to fetch dependency>
 
-if [ ! -d "${HOME}/zopen/boot/" ]; then
-  echo "boot directory does not exist.  Existing"
-  exit 4;
-fi
 
-for deps in $(find "${HOME}/zopen/boot/" -name ".env")
-do
-  echo "Found ${deps}"
-  depdir=$(dirname "${deps}")
-  cd $depdir
-  . ./.env
-  cd -
-done
+#below code fetches the ~/code/utils/bin path where the common.inc file is under its lib dir
+export bindir="$( cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd -P )"
+. "${bindir}/lib/common.inc"
+
+#depsPath has the details of the location of ports whose env files are consumed for sourcing 
+depsPath="$HOME/zopen/prod|$HOME/zopen/boot"
+
+# if the number of arguments passed to script is 0,
+# then , the dirs are fetched from zopen/prod and zopen/boot directory
+# then its env files are fetched and sourced, if they are not already sourced.
+if [ $# -eq 0 ]
+then
+     echo "buildenv path is not provided"
```

Comment:
I suggest changing the echo to "Source projects from <expand depsPath>"

Reply:
I have modified this!